tests/migration_test.go

```diff
@@ -54,7 +54,7 @@ const (
 	fedoraVMSize      = "256M"
 )
 
-var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system] VM Live Migration", func() {
+var _ = FDescribe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system] VM Live Migration", func() {
```

Comment:
Please remove the focus here.

Reply:
:man_facepalming: I really need to write checked for focus tests.
Thanks, removed!